src/Billing/Controllers/FreshdeskController.cs

```diff
@@ -77,7 +77,9 @@ public class FreshdeskController : Controller
 
                 foreach (var org in orgs)
                 {
-                    var orgNote = $"{org.Name} ({org.Seats.GetValueOrDefault()}): " +
+                    // Prevent org names from injecting any additional HTML
+                    var orgName = org.Name.Replace("<", string.Empty).Replace(">", string.Empty);
```

Comment:
suggestion:

An alternative approach we could consider is utilizing `HtmlUtility.Encode(org.Name)` from the `System.Web` namespace. Using that utility method should have the added benefit of taking care of addressing other HTML-reserved characters like `"`, `'`, and `&`. We could consider using `HtmlUtility.Encode` as more resilient since it would also handle any future additions or changes to reserved characters.
That said, the output of `HtmlUtility.Encode` would be different from this approach since it replaces characters with their encoded equivalents instead of just stripping them out. However, this could ensure that we aren't inadvertently discarding any non-malicious data!
Feel free to resolve this comment if you think the current approach suffices!

Reply:
I switched to HtmlEncode. This is a little less desirable from a display perspective since the `'` character will get encoded and I suspect it is somewhat common in org names, but overall I think is a safer solution.